codesrc/src/main/java/com/onehealth/core/processor/RequestProcessor.java

```diff
@@ -11,31 +10,31 @@ public void setRequest(req request) {
 		this.request = request;
 	}
 
```

Comment:
We can handle all the exceptions here or in Base Service or can throw it from each service as is done. Advantage to have it here is we will have it as a global block for handling all exceptions and wrapping it only to UI exceptions